boilerplate/app/theme/spacingDark.ts

```diff
@@ -0,0 +1,14 @@
+const SPACING_MULTIPLIER = 0.6
+
+// This is an example of how you can have different spacing values for different themes.
+export const spacing = {
```

Comment:
I think some spacing on light mode has been messed up by these changes, specifically the drawer menu and the visible icon for the password field (this is wrong on dark mode too).

I think the spacing on the demo screen between the header and first section also looks wrong, but I'm less sure it wasn't that way before.

<img width="430" alt="Screenshot 2024-08-05 at 2 23 51 PM" src="https://github.com/user-attachments/assets/d9d9c8e1-11a7-465a-b296-22cae752b8fb">


https://github.com/user-attachments/assets/432d9416-1734-465b-84ab-41e33713ad34

<img width="397" alt="Screenshot 2024-08-05 at 2 27 05 PM" src="https://github.com/user-attachments/assets/a891e731-7d1e-49a0-8f01-ec5613127633">

Reply:
Yeah, so the `theme/spacing.ts` is the ORIGINAL spacing file from the theme. `theme/spacingDark.ts` is a modified theme that has 1/2 the spacing of the light theme.

Current: https://github.com/infinitered/ignite/blob/master/boilerplate/app/theme/spacing.ts
Themed: https://github.com/infinitered/ignite/blob/mrickert/theming/boilerplate/app/theme/spacing.ts
Themed Dark (smaller): https://github.com/infinitered/ignite/blob/mrickert/theming/boilerplate/app/theme/spacingDark.ts

So the light mode is what ignite used to be and dark mode is a little tighter.

The problem of the submenus not showing in your first video has been fixed in a new push.